app/service/utils.go

```diff
@@ -163,6 +168,41 @@ func toSnakeCase(in string) string {
 	return string(out)
 }
 
+// GetResponseForRouteWithMockedDBAndUser executes a route for unit tests
```

Comment:
The functions dedicated to tests should be moved to a specific file (if we consider it as cross-package even, we should move in to "testhelpers" package). It bothers me to have monkey patching lib loaded in a file with the functions called in production. This lib is fragile and we should make sure it is not used by mistake.

Reply:
moved to app/servicetest